Source/WebKit/UIProcess/ios/WKMouseGestureRecognizer.mm

```diff
@@ -188,14 +188,15 @@ - (void)touchesCancelled:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event
 - (void)_hoverEntered:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event
 {
     _cancelledOrExited = NO;
-    _lastEvent = [self createMouseEventWithType:WebKit::WebEvent::MouseMove wasCancelled:NO];
-
+    
     if (_currentHoverEvent == nil && touches.count == 1 && [event isKindOfClass:NSClassFromString(@"UIHoverEvent")]) {
         _currentHoverEvent = event;
         _currentTouch = touches.anyObject;
         _lastLocation = [self locationInView:self.view];
         self.state = UIGestureRecognizerStateBegan;
```

Comment:
I think the self.state change needs to remain *after* creating _lastEvent, since setting the state is what triggers the gesture recognizer's delegate to read _lastEvent and dispatch it to the page (via `mouseGestureRecognizerChanged:` in `WKContentViewInteraction`).